devicetypes/smartthings/zwave-fan-controller.src/zwave-fan-controller.groovy

```diff
@@ -106,7 +106,7 @@ def fanEvents(physicalgraph.zwave.Command cmd) {
 	def value = (cmd.value ? "on" : "off")
 	def result = [createEvent(name: "switch", value: value)]
 	result << createEvent(name: "level", value: cmd.value == 99 ? 100 : cmd.value)
-	result << createEvent(name: "fanSpeed", value: cmd.value/33)
+	result << createEvent(name: "fanSpeed", value: cmd.value/33 as int)
```

Comment:
Would it be better to round that truncate?

Reply:
devices only ever send us multiples of 33, but I suppose it's more resilient